corrcal/test/corrcal_example.py

```diff
@@ -1,33 +1,38 @@
-import numpy, time
-import corrcal
-from data import DATA_PATH
```

Comment:
When I try and run the example, it fails because the data folder is in  a parent folder. Changing the import from data to ..data does not resolve the issue.

Hmm I remember us having issues with this

Reply:
This should be fixed in https://github.com/UKZN-Astronomy/corrcal/pull/21/commits/d357c3f38564de33b8d48da1eeacb977855ca70f.

The issue was that the data files were not included by `setuptools` when you run `pip install .`. Please uninstall `pip uninstall corrcal` and reinstall. Then, absolute import should work but make sure that you do not try to `import corrcal` from the local repository because it is usually at the top of the list in PYTHONPATH.